src/library_gl.js

```diff
@@ -7492,7 +7555,7 @@ var LibraryGL = {
 
 // Simple pass-through functions. Starred ones have return values. [X] ones have X in the C name but not in the JS name
 var glFuncs = [[0, 'finish flush'],
- [1, 'clearDepth clearDepth[f] depthFunc enable disable frontFace cullFace clear lineWidth clearStencil stencilMask checkFramebufferStatus* generateMipmap activeTexture blendEquation isEnabled*'],
```

Comment:
This one removes `glActiveTexture` from getting autogenerated, since in the `GL_STATE_CACHE` enabled case, we need to remember what the active texture was (at https://github.com/kripken/emscripten/pull/4898/files#diff-78a51faf7f8c4f62113aae3d5d4ae8eeR1384)